cogs/public.py

```diff
@@ -1,24 +1,31 @@
 import discord
 from discord.ext import commands
 
-
 class Public(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
         self.bot = bot
     
     @commands.hybrid_command(aliases=['av'])
-    async def avatar(self, ctx: commands.Context):
+    async def avatar(self, ctx: commands.Context, user: discord.User = None):
         """
         Display a user's avatar.
         """ 
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        user = user or ctx.author
+        embed = discord.Embed(color=discord.Color.blurple())
+        embed.set_author(name=f'{user.name}\'s avatar', icon_url=user.display_avatar.url)
+        embed.set_image(url=user.display_avatar.url)
+        await ctx.reply(embed=embed)
     
     @commands.hybrid_command()
-    async def banner(self, ctx: commands.Context):
+    async def banner(self, ctx: commands.Context, user: discord.User = None):
         """
         Display a user's banner.
         """ 
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        user = user or ctx.author
+        embed = discord.Embed(color=discord.Color.blurple())
+        embed.set_author(name=f'{user.name}\'s banner', icon_url=user.display_avatar.url)
```

Comment:
I think in this context, its better to use double quotation marks for the string, so that you don't have to escape the single quote.